cli/defenseclaw/commands/windows_native_uninstall.py

```
@@ -0,0 +1,760 @@
+        "$publisher=$sig.SignerCertificate.GetNameInfo("
+        "[Security.Cryptography.X509Certificates.X509NameType]::SimpleName,$false)};"
+        "$version=[Diagnostics.FileVersionInfo]::GetVersionInfo($args[0]).ProductVersion;"
+        "[pscustomobject]@{"
+        "Status=[string]$sig.Status;"
+        "Publisher=$publisher;"
+        "SignatureType=[string]$sig.SignatureType;"
+        "ProductVersion=[string]$version"
+        "}|ConvertTo-Json -Compress"
+    )
+    try:
+        completed = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)